configs/milvus.yaml

```diff
@@ -412,6 +412,10 @@ dataCoord:
       forceTrigger:
         minSize: 8 # The minmum size in MB to force trigger a LevelZero Compaction
         deltalogMinNum: 10 # the minimum number of deltalog files to force trigger a LevelZero Compaction
+  import:
+    filesPerPreImportTask: 2 # The maximum number of files allowed per pre-import task.
+    taskRetention: 10800 # The retention period in seconds for tasks in the Completed or Failed state.
```

Comment:
Please ensure that the  "taskRetention" here refers to the entire main task rather than the subtasks.